src/components/StaticBox.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { ReactNode, CSSProperties } from 'react'
+import { Paragraph, Separator, Text, XStack, YStack } from 'tamagui'
+import { styled } from 'tamagui'
+import { Image } from 'tamagui'
+import MyResponsiveLine from './StandardLineChart'
+
+interface IconProps {
+  source: string
+  width?: number
+  height?: number
+}
+
+const Icon: React.FC<IconProps> = ({ source, width = 16, height = 16, ...props }) => {
```

Comment:
We already have this Icon component, you can remove it and import Icon from components folder